lms/static/scripts/frontend_apps/utils/content-item.js

```diff
@@ -18,44 +18,3 @@
  *
  * @typedef {FileContent|URLContent} Content
```

Comment:
I left the file in for the type definitions but I don't think the compiler likes that

```
TS2306: File '/home/marcos/hypo/lms/lms/static/scripts/frontend_apps/utils/content-item.js' is not a module.
```

Reply:
In the client we add this to the bottom of some definition-only files as a workaround:

```js
// Make TypeScript treat this file as a module.
export const unused = {};
```